tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
Should you change this back to `es5`? Check the latest version supported natively by all platforms.